config/application.rb

```diff
@@ -12,6 +12,9 @@
 
 module SmartAnswers
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```

Comment:
Is your plan to switch on the new 5.2 framework defaults in a later PR?

Reply:
Mmmm, no. That's what rails did when I ran `rails app:update`. I thought it was ok anyway since I'll upgrade to rails 6 and this will become `config.load_defaults 6.0` but I can definitely change that to reflect the fact that this is `5.2`.